migration-engine/connectors/sql-migration-connector/Cargo.toml

```diff
@@ -7,6 +7,7 @@ version = "0.1.0"
 [dependencies]
 datamodel = {path = "../../../libs/datamodel/core"}
 datamodel-connector = {path = "../../../libs/datamodel/connectors/datamodel-connector"}
+sql-datamodel-connector = {path = "../../../libs/datamodel/connectors/sql-datamodel-connector"}
```

Comment:
I think it's unfortunate to have three imports for what is conceptually one thing (datamodel), but I don't have an alternative in mind yet, so let's go with this.